### ISSUE TITLE
Support angular 6 CLI 

### ISSUE THREAD

**Issue by almothafar** [zefoy/ngx-swiper-wrapper#185]
Using `ng add ngx-swiper-wrapper` should be helpful to add the module to project with full setup that this module need, and helpful for CLI commands during development and maybe staging, currently when I do this command I get error like:

```
Installed packages for tooling via npm.
Package "ngx-swiper-wrapper" was found but does not support schematics.
Error: Package "ngx-swiper-wrapper" was found but does not support schematics.
```

![image](https://user-images.githubusercontent.com/13887564/46740774-493d5e80-ccac-11e8-8c93-fc738a2387b7.png)

You can see an example of what it should do if you try this: https://www.telerik.com/kendo-angular-ui/components/charts/

**Comment by sconix** [comment]
I can have a look adding support when releasing version for Angular 7. Meanwhile PR's are accepted.

**Comment by sahilkatia** [comment]
@almothafar @sconix I am not sure but last month I created sample swiper application using Angular6 and it works fine for me.
Below is link to package.json
https://github.com/sahilkatia/mobileSwiper/blob/master/package.json

**Comment by almothafar** [comment]
@sahilkatia it is working in Angular 6 and I can confirm this for Angular 7 as well, I'm just talking about installing swiper using CLI `ng add ngx-swiper-wrapper`, this is minor for now, but this is not working anyway.

**Comment by sconix** [comment]
Yes the library works with Angular 6 and 7. This issue is about supporting the Angular CLI schematics so that one can take the library in use with cli commands.
